app/models/concerns/client_search.rb

```diff
@@ -89,5 +89,19 @@ def self.metaphone_search(where, field, text)
 
       where
     end
+
+    def self.name_search(arel_t, field, text, search_custom_names: true, custom_name_options: {})
+      query = arel_t[field].lower.matches(text)
+      return with_custom_name_search(query, field, text, **custom_name_options) if search_custom_names && custom_name_options.present?
```

Comment:
Any reason not to get rid of `search_custom_names`.  You default it to true but never pass in false.  Seems like checking for the options would be sufficient.
```suggestion
    def self.name_search(arel_t, field, text, custom_name_options: {})
      query = arel_t[field].lower.matches(text)
      return with_custom_name_search(query, field, text, **custom_name_options) if  custom_name_options.present?
```

Reply:
Sure, I can remove it. There's no use case for it at the moment